apps/synapse-portal-framework/src/components/cancercomplexity/CancerComplexityHeader.tsx

```diff
@@ -1,8 +1,5 @@
 import { Box, Typography, Link } from '@mui/material'
 import HeaderSearchBox from '../HeaderSearchBox'
-import Header from '../Header'
```

Comment:
Also release the new CCKP header from experimental mode in this release (per Jineta)